model.md

```diff
@@ -0,0 +1,66 @@
+# Pedigree conceptual model
+
+## Motivation
+
+Lack of interoperability of pedigree data between systems results in a substantial double-entry burden for already-overloaded nurses, genetic counselors, and support staff, and introduces the potential for serious errors.
+
+# Classes
+
+![image](https://user-images.githubusercontent.com/4251264/105409205-1c53b200-5bfe-11eb-9d77-ed5630138eaf.png)
+
+## Pedigree
+
+A `Pedigree` is collection of selected information about a family, including the individuals, relationships between them, and relevant health conditions.
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| proband | ID | optional | id of Individual that is the index case for the family |
+| consultand | ID | optional | id of Individual that is the focus of the current analysis |
+| collectedAt | Date | optional | the date the pedigree was collected, as ISO full or partial date, i.e. YYYY, YYYY-MM, or YYYY-MM-DD |
+| reasonCollected | Concept | optional | the health condition being investigated in the family; if any Individual has the `affected` property defined, it refers to this condition |
+
+## Individual
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| id | ID | required | logical id |
```

Comment:
Is ID a data type? If so, it isn't defined. If it isn't, should it not be a string type?

Reply:
Thanks for flagging. I clarified that this is indeed a string.